app/GraphQL/Subscription/Mutations/Prices/PriceMutation.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\GraphQL\Prices\Mutations;
+
+use Illuminate\Support\Facades\Auth;
+use Kanvas\Apps\Models\Apps;
+use Kanvas\Subscription\Prices\Actions\CreatePrice;
+use Kanvas\Subscription\Prices\Actions\UpdatePrice;
+use Kanvas\Subscription\Prices\DataTransferObject\Price as PriceDto;
+use Kanvas\Subscription\Prices\Models\Price as PriceModel;
+use Kanvas\Subscription\Prices\Repositories\PriceRepository;
+use Stripe\Price as StripePrice;
+use Stripe\Product as StripeProduct;
+
+class PriceMutation
+{
+    /**
+     * create.
+     */
+    public function create(array $req): PriceModel
+    {
+        $app = app(Apps::class);
+        $stripeProduct = StripeProduct::create([
```

Comment:
Do you know if Stripe has validation for not create duplicate records?

Reply:
Each product or price has a different ID in Stripe even if it has the same name and detail. That's why en is linked by the Stripe price_id.